docusaurus.config.js

```diff
@@ -1,127 +1,128 @@
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'KILT Documentation',
-  tagline: 'Documentation for KILT Node and SDK',
-  url: 'https://dev.kilt.io',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
-  organizationName: 'KILTprotocol', // the github org name. Will be used in the deploy step to clone the repository
-  projectName: 'docs', // the github project name. Will be used in the deploy step to clone the repository
-  themeConfig: {
-    navbar: {
-      title: '', // no title since the Name is already in the logo
-      logo: {
-        alt: 'KILT Logo',
-        src: 'img/logo_light.svg',
-        srcDark: 'img/logo_dark.svg',
-      },
-      items: [
-        {
-          to: 'docs/sdk/introduction',
-          activeBasePath: 'docs/sdk',
-          label: 'SDK',
-          position: 'left',
-        },
-        {
-          to: 'docs/chain/introduction',
-          activeBasePath: 'docs/chain',
-          label: 'Chain',
-          position: 'left',
-        },
-        {
-          to: 'docs/apps/introduction',
-          activeBasePath: 'docs/apps',
-          label: 'Apps',
-          position: 'left',
-        },
-        {
-          href: 'https://github.com/KILTprotocol/docs',
-          position: 'right',
-          className: 'header-github-link',
-          'aria-label': 'GitHub repository',
-        },
-      ],
-    },
-    footer: {
-      style: 'dark',
-      links: [
-        {
-          title: 'Community',
-          items: [
-            {
-              label: 'Element / Matrix',
-              href: 'https://matrix.to/#/#kilt-general:matrix.org',
-            },
-            {
-              label: 'Discord',
-              href: 'https://discord.gg/hX4pc8rdHS',
-            },
-            {
-              label: 'Telegram',
-              href: 'https://t.me/KILTProtocolChat',
-            },
-            {
-              label: 'Twitter',
-              href: 'https://twitter.com/Kiltprotocol',
-            },
-            {
-              label: 'Clan KILT (Unofficial)',
-              href: 'https://clankilt.io',
+    title: 'KILT Documentation',
```

Comment:
I think this is only a style change? We normally indent with 2 spaces. Could you revert this change?